backend/contact/routes/support_request/person_test.go

```diff
@@ -0,0 +1,41 @@
+package support_request
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const personPayload = `{
+	"address": "Sofia",
```

Comment:
@imilchev Let's add a field `comment` as it's a mandatory part of the payload now